Dockerfile.dev

```diff
@@ -6,19 +6,20 @@ RUN \
     curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
     && apt-get update \
     && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-        # Additional library needed by some tests and accordingly by VScode Tests Discovery
-        bluez \
-        libudev-dev \
-        libavformat-dev \
-        libavcodec-dev \
-        libavdevice-dev \
-        libavutil-dev \
-        libswscale-dev \
-        libswresample-dev \
-        libavfilter-dev \
-        libpcap-dev \
-        libturbojpeg0 \
-        git \
+    # Additional library needed by some tests and accordingly by VScode Tests Discovery
+    bluez \
+    libudev-dev \
+    libavformat-dev \
+    libavcodec-dev \
+    libavdevice-dev \
+    libavutil-dev \
+    libswscale-dev \
+    libswresample-dev \
+    libavfilter-dev \
+    libpcap-dev \
+    libturbojpeg0 \
+    git \
+    cmake \
```

Comment:
@justinedelson can you keep the indentation as it was in the original file? 
This makes it easier to read and understand.
Thanks

Reply:
Happy to do so, but... VSCode automatically reformatted this on save and it appeared to me that this reformatting was based on the settings in `.devcontainer/devcontainer.json`